src/VisualStudio/VisualBasic/Impl/Snippets/SnippetFunctions/SnippetFunctionArgumentValue.vb

```diff
@@ -0,0 +1,23 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.VisualStudio.LanguageServices.Implementation.Snippets
+Imports Microsoft.VisualStudio.Text
+
+Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Snippets.SnippetFunctions
+    Friend NotInheritable Class SnippetFunctionArgumentValue
+        Inherits AbstractSnippetFunctionArgumentValue
+
+        Public Sub New(snippetExpansionClient As SnippetExpansionClient, subjectBuffer As ITextBuffer, parameterName As String, parameterKey As SymbolKey)
+            MyBase.New(snippetExpansionClient, subjectBuffer, parameterName, parameterKey)
+        End Sub
+
+        Protected Overrides ReadOnly Property FallbackDefaultLiteral As String
+            Get
+                Return "Nothing"
+            End Get
+        End Property
```

Comment:
```suggestion
        Protected Overrides ReadOnly Property FallbackDefaultLiteral As String = "Nothing"
```

Nit: i know this has different semantics.  But seeing 5 lines to return a cosntant just pains my soul.  Having this instance be slightly larger because it has a field pointing at at a constnat is fine with me :)

Reply:
➡️ Visual Basic opts for a longer syntax here, which this code uses. Request for a shorter syntax is dotnet/vblang#61.